test_isort.py

```diff
@@ -2592,3 +2593,20 @@ def test_monkey_patched_urllib():
         # Previous versions of isort monkey patched urllib which caused unusual
         # importing for other projects.
         from urllib import quote  # noqa: F401
+
+
+@pytest.mark.parametrize('enabled', (False, True))
+def test_safety_excludes(tmpdir, enabled):
+    from isort import settings, main
```

Comment:
Can you move the import to the top level, please?

Reply:
Sure thing.